.github/workflows/repository.yml

```diff
@@ -32,24 +32,23 @@ jobs:
         run:  tests/file-extensions.sh
       - name: Validate Images
         run: bundle exec ruby ./tests/validate-images.rb
-      - name: Validate URL/Domain reachability
-        run: bundle exec ruby ./tests/validate-urls.rb
-        continue-on-error: true
```

Comment:
Any reason why this check has been dropped?

Reply:
Whoops